platforms/web/web.js

```diff
@@ -0,0 +1,158 @@
+"use strict";
+
+const Bluebird = require("bluebird");
+const { ButtonsListMessage, Button, TextMessage, ButtonsMessage} = require("../generics");
+const WebMessage = require("../../models/web.model");
+
+
+function getTextMessage(message) {
+  let messageData = {
+    message,
+    buttons : []
+  };
+
+  return messageData;
+}
+
+function getButton(button) {
+  let messageData = {
+    message : null,
+    buttons : [
+      {
+        type: button.type,
+        value: button.value,
+        text: button.text
+      }
+    ]};
+
+  return messageData;
+}
+
+function getListButton(buttonList) {
+  let messageData = {
+    message : buttonList.text,
+    buttons : []
+  };
+
+  if (buttonList.attachments.buttons) {
+    for(let i = 0; i < buttonList.attachments.buttons.length; i++) {
+      let but = buttonList.attachments.buttons[i];
+      messageData.buttons.push(
+        {
+          type: but.type,
+          value: but.value,
+          text: but.text
+        });
+    }
+  }
+
+  return messageData;
+}
+
+function sendToClient(res, messageData) {
+  return res.status(200).send(messageData);
+}
+
+
+
+function send(retour, data) {
+  let senderId = retour.senderId;
+  let flagUseRes = true;
+
+  if (typeof retour === "string") {
+    senderId = retour;
+    retour = null;
+    flagUseRes = false;
+  }
+
+  data = data.length == null || typeof data === "string" ? [data] : data;
```

Comment:
So ... data can be a boolean or an array in this case. I think there is a mistake about `data.length == null` which return a boolean. Because next line is about an array